live/app/routes/board.js

```diff
@@ -16,12 +16,11 @@ export default BaseRoute.extend(AuthenticatedRouteMixin, {
         if (user.get('organizations.length') <= 0) {
           return this.transitionTo('compte');
         }
-
         const organization = user.get('organizations.firstObject');
-
         return RSVP.hash({
           organization,
-          snapshots: organization.get('snapshots').reload()
+          snapshots: organization.get('snapshots').reload(),
```

Comment:
à partir de la ligne 14, tout déplacer dans le after-model

Reply:
les catch vont dans l'action error

Reply:
cf https://guides.emberjs.com/v2.16.0/routing/loading-and-error-substates/#toc_the-code-error-code-event

Reply:
Le model est créé entre ces lignes.
Il semble que le model (utilisé dans le template) ne peut être modifié dans le afterModel.
Par conséquent, en l'état, je ne vois pas comment utiliser l'afterModel (et s'il faudrait l'utiliser ou non)